### PR TITLE
Update ome-faq.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a couple of more questions to the FAQ section regarding send as and Full access

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3940

File: microsoft-365/compliance/ome-faq.md [hunk 1]
```diff
@@ -175,6 +175,11 @@ Admins can set up a mail flow rule to remove encryption from outgoing mail. You
 
 Not at this time.
 
+
+## Can I send as a shared mailbox and encrypt emails?
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.md:179]:
```suggestion
## Can I send an encrypted email from a shared mailbox?
```

File: microsoft-365/compliance/ome-faq.md [hunk 2]
```diff
@@ -209,6 +215,10 @@ There are currently two known limitations:
    ```powershell
    Add-MailboxPermission -Identity support@contoso.onmicrosoft.com -User ayla@contoso.com -AccessRights FullAccess -AutoMapping $true
    ```
+   
+ ## Can I open encrypted messages sent to another user mailbox with Fullaccess?
+
+Users can open encrypted messages as long as they are given direct access and automapping is turned ON. We don't support if the access to granted via an email enabled security group.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.md]:
```suggestion
Users can open encrypted messages as long as they are given direct access and automapping is turned ON. Access is not allowed if the access is granted via an email-enabled security group.
```

File: microsoft-365/compliance/ome-faq.md [hunk 3]
```diff
@@ -209,6 +215,10 @@ There are currently two known limitations:
    ```powershell
    Add-MailboxPermission -Identity support@contoso.onmicrosoft.com -User ayla@contoso.com -AccessRights FullAccess -AutoMapping $true
    ```
+   
+ ## Can I open encrypted messages sent to another user mailbox with Fullaccess?
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.md]:
```suggestion
 ## Can I open encrypted messages sent to another user's mailbox with Fullaccess?
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

File: microsoft-365/compliance/ome-faq.md [hunk 4]
```diff
@@ -200,6 +205,7 @@ There are currently two known limitations:
 
 - We don't support assignment through an email enabled security group. We only support access provided by direct user assignment to the shared mailbox and that automapping is enabled for Exchange Online. Automapping is enabled by default for Exchange Online.
 
+
```
Comment (KCCross) [inline microsoft-365/compliance/ome-faq.md]: Delete this line

File: microsoft-365/compliance/ome-faq.md [hunk 5]
```diff
@@ -175,6 +175,11 @@ Admins can set up a mail flow rule to remove encryption from outgoing mail. You
 
 Not at this time.
 
+
```
Comment (KCCross) [inline microsoft-365/compliance/ome-faq.md]: Delete this line

**Comment by KCCross** [comment]
Removed the extra lines and i'll merge.

**Pull request merged**